.devcontainer/images/.claude/commands/infra/terraform.md

```
@@ -0,0 +1,324 @@
+    files: ["terragrunt.hcl"]
+    tool: "terragrunt"
+
+  opentofu:
+    files: ["*.tf", ".terraform-version"]
+    check: "tofu version"
+    tool: "tofu"
+```
+
+**Output:**
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)